spec/mongo/server/app_metadata_spec.rb

```diff
@@ -66,7 +66,9 @@
         end
 
         it 'truncates the document' do
-          expect(app_metadata.send(:ismaster_bytes)).to be_a(String)
+          expect(
+            app_metadata.validated_document.to_bson.to_s.size
+          ).to be < described_class::MAX_DOCUMENT_SIZE
```

Comment:
Can this constant be replaced by the literal value (512?) OR the assertion made additionally that the constant is no greater than 512, to maintain the meaningfulness of this test if document size increases.